docker-compose.yml

```diff
@@ -71,9 +71,9 @@ services:
     environment:
       KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: zookeeper1:2181,zookeeper2:2181,zookeeper3:2181
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092
-      KAFKA_DEFAULT_REPLICATION_FACTOR: 2        
```

Comment:
It's better to keep this config. So, we can ensure all topics created with the default configuration have two replicas, which can help with data availability.